src/components/PageBlocks/Wrap/Plain.vue

```diff
@@ -11,12 +11,22 @@
         <div
           v-if="!headerSet"
         >
-          <h5
+          <div
             v-if="block.title"
-            class="text-truncate mb-0"
+            class="d-flex justify-content-between align-content-center"
           >
-            {{ block.title }}
-          </h5>
+            <h5
+              class="text-truncate mb-0"
+            >
+              {{ block.title }}
+            </h5>
+
+            <font-awesome-icon
+              v-if="block.options.refreshRate >= 5"
+              :icon="['fa', 'sync']"
+              class="h6 text-light mb-0"
```

Comment:
text-secondary, we should also consider turning this into a button that triggers a manual refresh